src/object_async/hello_async.cpp

```diff
@@ -206,7 +205,7 @@ struct AsyncHelloWorker : Nan::AsyncWorker // NOLINT to disable cppcoreguideline
         }
     }
 
-    std::unique_ptr<std::string> result_;
+    std::unique_ptr<std::string> result_{};
```

Comment:
Can you use `make_unique` instead?

Reply:
I wondered about this. I think `std::unique_ptr<std::string> result_{}` may be creating an object that is an invalid/nullptr. Whereas using `make_unique` will create a valid pointer pointing toward an empty string. Let me give that latter a shot since I think that is ideal.

Reply:
Worked great, thanks for this suggestion @alliecrevier